README.md

```diff
@@ -14,20 +14,20 @@ This docker image is based on alpine
 * alpine is based on musl libc and busybox. As oracle java depends on glibc, it
   doesn't work smoothely. With https://github.com/sgerrand/alpine-pkg-glibc this
   can be workarounded to a certain way but in our case not sufficient (conflicting dependencies).
-* Current SNAP Version (7.0.2) is recommended to be run with JAVA 8.
+* Current SNAP Version (8.0.0) is recommended to be run with JAVA 8.
```

Comment:
This might be misleading, as it is a quote from the website. It was recommended for 7.0.2, so I would leave it as is.

Reply:
ok reverted